services/client/src/components/event/time-to-event.js

```diff
@@ -2,29 +2,56 @@ import React from "react";
 import PropTypes from "prop-types";
 import moment from "moment";
 
+const DateOfEvent = () => {
+  const day = moment.utc(day);
+
+  return (
+    <React.Fragment>
+      <style jsx>{`
+        span {
+            color: #989fa5;
+            font-style: italic;
+          }
+        `}</style>
+      <span>
+        {day}
+      </span>
+    </React.Fragment>
+  )
+}
+
 const TimeToEvent = ({ startTime }) => {
   const now = moment();
   const eventTime = moment.utc(startTime);
   const duration = moment.duration(eventTime.diff(now));
+  const date = eventTime;
 
   return (
     <React.Fragment>
       <style jsx>{`
-        span {
+        #when {
```

Comment:
Changing the span tag here will remove the styling of the `<span>` as `when` does not appear to be used. Is this intended?

Reply:
`when` is being used, its the styling for the time until the next event.